route/dispatch.go

```diff
@@ -0,0 +1,22 @@
+package route
+
+import "github.com/Dieterbe/go-metrics"
+
+// DispatchNonBlocking will dispatch in to buf.
+// if buf is full, will discard the data
+func dispatchNonBlocking(buf chan []byte, in []byte, gauge metrics.Gauge) {
+	select {
+	case buf <- in:
+		gauge.Inc(1)
+	default:
```

Comment:
wouldn't it make sense to count the dropped ones?

Reply:
Yeah seems like a good thing to instrument